README.md

```diff
@@ -128,3 +128,40 @@ Below is an example command
 ```sh
 docker run -v $PWD:$PWD -v /usr/local/bin:/usr/local/bin -w $PWD ghcr.io/gem5/gcn-gpu:v24-0 gem5-vega-se gem5/configs/example/apu_se.py -n 3 -c square
 ```
+
+## Note on running GPU FS
+
+You need not use docker to run simulations in GPU FS mode. You still need docker to compile applications like square. We will be simulating AMD's latest GPU, MI300, in FS mode
+Below is an example command
+
+```sh
+# To compile square
+cd /workspaces/gem5-bootcamp-env
+wget https://cs.wisc.edu/~ramadas/gem5-isca24/gpu-fs/Makefile
```

Comment:
Is there a reason you have to wget these things? Why not just copy them into materials/isca24

Reply:
Sounds good. Will do that.